README.md

```diff
@@ -100,6 +100,34 @@ The second is optional "evidence" that's published alongside the
 assertions. In the future this could for instance be used for
 serialized DNSSEC proofs.
 
+We can also create an assertion request that roughly matches an existing X.509
```

Comment:
Just to avoid having to clarify what 'roughly matches' means:
```suggestion
We can also create an assertion request derived from an existing X.509
```